src/uct/tcp/tcp_ep.c

```diff
@@ -1589,18 +1589,69 @@ static void uct_tcp_ep_post_put_ack(uct_tcp_ep_t *ep)
     ep->flags &= ~UCT_TCP_EP_FLAG_PUT_RX_SENDING_ACK;
 }
 
+static inline ucs_status_t
+uct_tcp_ep_am_short_sendv(uct_tcp_ep_t *ep, uct_tcp_iface_t *iface,
+                          uct_tcp_am_hdr_t *hdr, uint64_t header,
+                          const void *payload, size_t count, struct iovec *iov,
```

Comment:
please add the comment that that `count` is either bytes or number of IOVs passed in `payload`
just to make it clear